<launch>

<node pkg="image_transport" type="republish" name="republish" args="compressed">
	<remap from="/in" to="/axis/image_raw"/>
	<remap from="/out" to="/axis/repub/image_raw"/>
</node>
  
</launch>

